lib/modem_jwt/modem_jwt.c

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <string.h>
+#include <zephyr.h>
+#include <stdlib.h>
+#include <stdio.h>
+#include <modem/at_cmd.h>
+#include <modem/modem_jwt.h>
+
+#define JWT_CMD_TEMPLATE "AT%%JWT=%d,%u,\"%s\",\"%s\",%d,%d\r\n"
```

Comment:
Newline characters are not required in the commands, null termination is sufficient